tests/test_algorithms.py

```diff
@@ -767,12 +767,6 @@ def test_okp_ed25519_should_reject_non_string_key(self):
         with open(key_path("testkey_ed25519.pub")) as keyfile:
             algo.prepare_key(keyfile.read())
 
-    def test_okp_ed25519_should_accept_unicode_key(self):
```

Comment:
can you please describe why this test is removed?

Reply:
It's a duplicate of

https://github.com/jpadilla/pyjwt/blob/ba726444a6cee75af59feb8ea08294d0ac89bedb/tests/test_algorithms.py#L758-L765

L764-765

In fact I think it's better to move the two valid checks from `test_okp_ed25519_should_reject_non_string_key` to this removed `test_okp_ed25519_should_accept_unicode_key`. Tell me if you want me to do it this way